aries-site/src/themes/aries.js

```diff
@@ -204,6 +212,18 @@ export const aries = deepMerge(hpe, {
       xxlarge: '166px',
     },
   },
+  tab: {
```

Comment:
Do we want to explicitly set `tab.active.color` to 'text' in this theme? It is the default in Grommet's base.js, but I wonder if it might be beneficial to make that clear here?

Reply:
I am open to that for clarity, but is common convention only to define things that alter from grommet base.js?

Reply:
The custom theme is using base.js by default, so it may be redundant to declare styles that already exist in base.js unless they are altered.

Reply:
Yes, it would be redundant. Fine with it as is.